aea/configurations/base.py

```diff
@@ -1447,19 +1510,21 @@ def component_type(self) -> ComponentType:
     @property
     def json(self) -> Dict:
         """Return the JSON representation."""
-        return {
-            "name": self.name,
-            "author": self.author,
-            "version": self.version,
-            "license": self.license,
-            "aea_version": self.aea_version,
-            "fingerprint": self.fingerprint,
-            "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
-            "dependencies": self.dependencies,
-            "description": self.description,
-            "path_to_contract_interface": self.path_to_contract_interface,
-            "class_name": self.class_name,
-        }
+        return OrderedDict(
+            {
+                "name": self.name,
+                "author": self.author,
+                "version": self.version,
+                "description": self.description,
+                "license": self.license,
+                "aea_version": self.aea_version,
+                "fingerprint": self.fingerprint,
+                "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
+                "dependencies": self.dependencies,
```

Comment:
In spirit of above, would have, "class_name", "path_to_contract_interface", "dependencies"